### PR TITLE
Fix dissolve match modal

### PR DESCRIPTION
**Pull request by Jonasdoubleyou**

### CODE REVIEW

File: src/modals/DissolveMatchModal.tsx [hunk 1]
```diff
@@ -41,7 +43,7 @@ const DissolveMatchModal: React.FC<DissolveModalProps> = ({ showDissolveModal, o
                         </Modal.Body>
                         <Modal.Footer>
                             <Row space={space['1']}>
-                                <Button isDisabled={!reason} onPress={() => onPressDissolve(reason)}>
```
Comment (Jonasdoubleyou) [inline src/modals/DissolveMatchModal.tsx:44]: Need to check whether the user can actually request the a new match

File: src/modals/DissolveMatchModal.tsx [hunk 2]
```diff
@@ -56,24 +58,35 @@ const DissolveMatchModal: React.FC<DissolveModalProps> = ({ showDissolveModal, o
                         <Modal.Header>Neue Lernpartner:in anfordern</Modal.Header>
                         <Modal.Body>
                             <Text>
-                                Möchest du mit einem:r neuen Lernpartner:in verbunden werden? Beachte dabei, dass du wieder einige Zeit warten musst bis wir
-                                jemanden für dich finden können.
+                                {userType === 'pupil' && (
+                                    <>
+                                        Möchest du mit einem:r neuen Lernpartner:in verbunden werden? Beachte dabei, dass du wieder einige Zeit warten musst bis
+                                        wir jemanden für dich finden können.
```
Comment (timidak) [inline src/modals/DissolveMatchModal.tsx:64]: I would move this to the translated strings in i18next

File: src/modals/DissolveMatchModal.tsx [hunk 3]
```diff
@@ -56,24 +58,35 @@ const DissolveMatchModal: React.FC<DissolveModalProps> = ({ showDissolveModal, o
...
+                                    </>
+                                )}
+                                {userType === 'student' && (
+                                    <>
+                                        Wir würden uns wahnsinnig freuen, wenn du eine:n neue:n Schüler:in unterstützen möchtest. Bei uns warten sehr viele
+                                        Kinder auf individuelle Unterstützung und es wäre der absolute Hammer, wenn du uns bei unserer Mission weiter
+                                        unterstützst!
```
Comment (timidak) [inline src/modals/DissolveMatchModal.tsx:71]: > I would move this to the translated strings in i18next

Same here

**Review by timidak — APPROVED** [review]
LGTM, just the translated texts should be provided to i18next,

**Pull request closed without merging**